styx-common/src/main/java/com/spotify/styx/util/CachedSupplier.java

```diff
@@ -18,10 +18,9 @@
  * -/-/-
  */
 
-package com.spotify.styx;
+package com.spotify.styx.util;
```

Comment:
Something from Guava sounds more reliable. What do you think, @rouzwawi ? Did we have some problem then decided to have our own cached solution?